packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,46 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export async function getSchedulerGranularity({ host }) {
+  let schedGran
+  try {
+    schedGran = await this.getXapi(host).getField('host', host.id, 'sched_gran')
+  } catch (e) {
+    schedGran = e.code
```

Comment:
- We should only catch the error `UNKNOWN_METHOD`  to not ignore other errors which can give information about the dysfunction of this method

- `getSchedulerGranularity` returns the value of the granularity instead of the error